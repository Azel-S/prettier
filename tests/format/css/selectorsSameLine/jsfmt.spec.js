run_spec(__dirname, ["css"], {selectorsSameLine: false});
run_spec(__dirname, ["css"], {selectorsSameLine: true});