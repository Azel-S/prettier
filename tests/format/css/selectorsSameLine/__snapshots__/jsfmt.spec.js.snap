// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`selectorsSameLine.css - {"selectorsSameLine":false} format 1`] = `
====================================options=====================================
parsers: ["css"]
printWidth: 80
selectorsSameLine: false
                                                                                | printWidth
=====================================input======================================
.h1,    .h2 {
    margin: 0%;
}
=====================================output=====================================
.h1,
.h2 {
  margin: 0%;
}

================================================================================
`;

exports[`selectorsSameLine.css - {"selectorsSameLine":true} format 1`] = `
====================================options=====================================
parsers: ["css"]
printWidth: 80
selectorsSameLine: true
                                                                                | printWidth
=====================================input======================================
.h1,    .h2 {
    margin: 0%;
}
=====================================output=====================================
.h1, .h2 {
  margin: 0%;
}

================================================================================
`;
